src/style.css

```diff
@@ -0,0 +1,58 @@
+body {
+  margin: 0;
+}
+
+#header {
+  margin: 0 auto;
```

Comment:
these styles are repeated many times
```
  width: 300px;
  height: 100px;
  margin: 0 auto;
```
 you can write them like this
```
#header, .important, ... {
  width: 300px;
  height: 100px;
  margin: 0 auto;
}
```
also, you can write the same hover